### PR TITLE
Personality Captions interactive mode fix

### PR DESCRIPTION
**Pull request by EricMichaelSmith**
**Patch description**
Implements the proposed fix at https://github.com/facebookresearch/ParlAI/issues/2951 to get interactive mode for Personality Captions to work with CUDA enabled.

**Testing steps**
```
python projects/personality_captions/interactive.py -mf models:personality_captions/transresnet/model
python projects/personality_captions/interactive.py -mf models:personality_captions/transresnet/model --no-cuda
```

### CODE REVIEW

File: projects/personality_captions/transresnet/modules.py [hunk 1]
```diff
@@ -399,9 +399,9 @@ def choose_best_caption(
         for img_index in range(len(context_encoded)):
             context_encoding = context_encoded[img_index : img_index + 1, :]
             scores = torch.mm(
-                candidates_encoded[img_index]
+                candidates_encoded[img_index].to(context_encoding.device)
```
Comment (klshuster) [inline projects/personality_captions/transresnet/modules.py:402]: Nit: I believe you can just do `.to(context_encoding`

File: projects/personality_captions/interactive.py [hunk 2]
```diff
@@ -275,8 +275,6 @@ def setup_interactive():
         )
     opt['task'] = 'parlai.agents.local_human.local_human:LocalHumanAgent'
     opt['image_mode'] = 'resnet152'
-    opt['no_cuda'] = True
```
Comment (klshuster) [inline projects/personality_captions/interactive.py:278]: I'm happy with this but was it contributing to the bug?

File: projects/personality_captions/transresnet/modules.py (hunk 1 above)
Comment (stephenroller) [reply projects/personality_captions/transresnet/modules.py:402]: rly? that's new to me.

Comment (klshuster) [reply projects/personality_captions/transresnet/modules.py:402]:
```python
In [1]: import torch

In [2]: torch.__version__
Out[2]: '1.4.0'

In [3]: a = torch.LongTensor([1, 2, 3]).cuda()

In [4]: a
Out[4]: tensor([1, 2, 3], device='cuda:0')

In [5]: b = torch.LongTensor([4, 5, 6]).to(a)

In [6]: b
Out[6]: tensor([4, 5, 6], device='cuda:0')

```

Comment (EricMichaelSmith) [reply projects/personality_captions/transresnet/modules.py]: Yes, changed

File: projects/personality_captions/interactive.py (hunk 2 above)
Comment (klshuster) [reply projects/personality_captions/interactive.py:278]: totally fine with the change, can't remember myself why it was in there in the first place

**Pull request merged**